### DISCUSSION TITLE
能否增加已失效视频的弹幕下载？

### DISCUSSION THREAD

**Discussion by ghost** [the1812/Bilibili-Evolved#3753] [功能建议 / Ideas]
之前一直用Biliplus查找、下载已失效视频的弹幕，比如以前个人上传的电影剧集，这几个月从Biliplus下载的弹幕都是空的显示【本视频弹幕功能已被关闭】，CID法也失效了，想来问问大家有没有别的途径……

**Comment by the1812** [comment]
这能从哪变出来。。以前恢复失效视频封面做一个方法就被封一个

**Reply by ghost** [reply]
唉那看来是彻底找不到那些弹幕了，放弃
